lib/openstack_api/openstack_identity.py

```diff
@@ -136,3 +141,29 @@ def find_user_all_domains(
 
         with self._connection_cls(cloud_account) as conn:
             return conn.identity.find_user(user_identifier, ignore_missing=True)
+
+    def get_project_email(self, project: Project) -> Union[str, None]:
+        """
+        Returns the contact email of a project
+        :param project: The project to obtain the email from
+        :return: The found email or None
+        """
+        found_email = None
+        for tag in project["tags"]:
+            if "@" in tag:
+                found_email = tag
+                break
+        return found_email
+
+    def find_project_email(self, cloud_account: str, project_identifier: str):
+        """
+        Returns the contact email of a project
+        :param cloud_account: The clouds entry to use
+        :param project_identifier: The name or Openstack ID for the project
+        :return: The found email or None
+        """
+        found_email = None
+        project = self.find_project(cloud_account, project_identifier)
```

Comment:
Should this be find_mandatory_project ? As it should always be a valid project specified

Reply:
I will double check there are no bugged FIPs, otherwise yes it should be.

Reply:
There are some, so it will have to be find project.